validator/accounts/v2/wallet_create.go

```diff
@@ -40,15 +39,12 @@ func CreateAndSaveWalletCli(cliCtx *cli.Context) (*wallet.Wallet, error) {
 	}
 
 	dir := createWalletConfig.WalletCfg.WalletDir
-	kmKind := createWalletConfig.WalletCfg.KeymanagerKind
-	accountsPath := filepath.Join(dir, kmKind.String())
-	ok, err := fileutil.HasDir(accountsPath)
+	ok, err := fileutil.HasDir(dir)
```

Comment:
I feel like in this particular case it would be better to name the variable `hasDir` instead of `ok`. I know this is a standard Go pattern, but checking `if ok` and then returning an error looks confusing.